stumpy/core.py

```diff
@@ -1217,6 +1217,13 @@ def mass_absolute(Q, T, T_subseq_isfinite=None, p=2.0):
     p : float, default 2.0
         The p-norm to apply for computing the Minkowski distance.
 
+    query_idx : int, default None
+        This is the index position along the time series, `T`, where the query
+        subsequence, `Q`, is located. `query_idx` should be set to None if `Q`
+        is not a subsequence of `T`. If `Q` is a subsequence of `T`, provding
+        this argument is optional. If provided, the precision of computation
+        can be slightly improved.
```

Comment:
@seanlaw 
How do you feel about the following description?

```suggestion
        can be slightly improved by setting the distance between a subsequence to self to 0.0 when all the values of the subsequence are finite.
```

Reply:
I don't even think we should say "slight improved". We should state concrete facts:

> If provided, the distance between `Q` and `T[query_idx : query_idx + m]` will automatically be set to zero.

Reply:
Correct! I will revise the description.